.ci/Jenkinsfile

```diff
@@ -186,7 +186,7 @@ void scheduleJob(jobs, String image, buildArgs, originalTags) {
     // buildArgs: The build args matrix
     // originalTags: The original tags
 
-    String markers = 'not daily'
+    String markers = 'not daily and not doctest'
```

Comment:
we don't wanna run doctests for each PR? what if your PR changes a docstring?

Reply:
It's being migrated to the `tests` area, see `tests/test_docs.py` and there's a new doctest marker.

Reply:
the doctests do run on every pr, this is the default args in the jenkins file

Reply:
Yep, so I copied the tests into `/tests` folder and added a new tag. Jenkins does something bespoke for doctests, I removed that and made it normal tests. However, I don't want to muck with Jenkins build to switch it, so they don't do that run